tox.ini

```diff
@@ -22,6 +22,7 @@ deps =
 
 commands =
      tests-!windows-!mac: sh -c "pybamm_install_jax"  # install jax, jaxlib for ubuntu
+     unit-!windows-!mac: sh -c "pybamm_install_jax"  # install jax, jaxlib for ubuntu
```

Comment:
This ensures that every `jax` related thing in PyBaMM is unit tested on all Python versions and not just `3.9` (during coverage).